### DISCUSSION TITLE
Make package.json friendly

### DISCUSSION THREAD

**Discussion by cncolder** [exceljs/exceljs#2521] [Ideas & Proposals]
1. Move `@types/node` into devDependencies.
2. Do not pin jszip version `"jszip": "^3.1.3"`.
3. Consider replace moment with [dayjs](/iamkun/dayjs). It's too heavy since we only parse time string.
4. add `browser` field.

**Comment by Siemienik** [comment]
@cncolder  Could you open PR with these changes?

**Comment by A1rPun** [comment]
1. Is done already
2. Pinning on major version is good practice, why removing this caret for `jszip` specifically?
3. This is a must. I wanted to make use of this library until I saw `moment` in the `package.json` which is [33% of your current codebase](https://bundlephobia.com/result?p=exceljs@1.14.0)!
4. Is done already
